tests/test_xdsl_op_correctness.py

```diff
@@ -41,16 +41,16 @@ def test_u_plus1_conversion():
     assert np.isclose(norm1, norm2, atol=1e-5, rtol=0)
     assert np.isclose(norm1, 23.43075, atol=1e-5, rtol=0)
 
-@pytest.mark.xfail(reason="Needs a fix in offsets")
+
 def test_u_and_v_conversion():
     # Define a simple Devito Operator
     grid = Grid(shape=(3, 3))
-    u = TimeFunction(name='u', grid=grid, time_order=2)
-    v = TimeFunction(name='v', grid=grid, time_order=2)
+    u = TimeFunction(name='u', grid=grid, time_order=3)
```

Comment:
why time_order =3?

Reply:
Just a leftover from exploring; I tried examples that were doing in-place when using time_order 2.
Switched this example back to time_order 2.